internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -658,26 +658,48 @@ func convertSubscribe(s manifest.SubscribeConfig, accountID, region, app, env, s
 	}
 	var subscriptions template.SubscribeOpts
 	for _, sb := range s.Topics {
-		ts := convertTopicSubscription(sb, sqsEndpoint.URL, accountID, app, env, svc)
+		ts, err := convertTopicSubscription(sb, sqsEndpoint.URL, accountID, app, env, svc)
+		if err != nil {
+			return nil, err
+		}
 		subscriptions.Topics = append(subscriptions.Topics, ts)
 	}
 	subscriptions.Queue = convertQueue(s.Queue)
 	return &subscriptions, nil
 }
 
-func convertTopicSubscription(t manifest.TopicSubscription, url, accountID, app, env, svc string) *template.TopicSubscription {
+func convertTopicSubscription(t manifest.TopicSubscription, url, accountID, app, env, svc string) (
```

Comment:
What do you think of passing a pointer instead since a `TopicSubscription` can be a fairly large object?
```suggestion
func convertTopicSubscription(sub *manifest.TopicSubscription) (
```
I can't tell if it's my lack of sleep 😅 but the remaining parameters here don't seem to be used in the body of the fn

Reply:
lol sorry obviously i my lack is worse :(